data/shortcodes.md

```diff
@@ -128,7 +129,13 @@
 | 0856      | sgb                  | Stadtgeschichte Basel                                                                                                                                                                       |               |       |
 | 0857      | symblight            | Symbolism of Light in Early Christian Sacred Buildings in the Swiss Alpine Valleys                                                                                                          |               |       |
 | 0858      | luminosity           | Luminosity of the East                                                                                                                                                                      |               |       |
-| 0859      | zhdk-smre            | Musikförderung über den Äther (Die Kompositionsaufträge der SRG und ihre Wirkungen auf das Schweizer Musikleben (1931-2021))                                                               |               |       |
+| 0859      | zhdk-smre            | Musikförderung über den Äther (Die Kompositionsaufträge der SRG und ihre Wirkungen auf das Schweizer Musikleben (1931-2021))                                                                |               |       |
+| 085A      | atlasdisciplines     | Interactive Historical Atlas of the Disciplines                                                                                                                                             |               |       |
+| 085B      | [free to use]        | 
+| 085C      | [free to use]        | 
+| 085D      | [free to use]        | 
+| 085E      | [free to use]        | 
+| 085F      | [free to use]        | 
```

Comment:
To be precise, you should add the `| ` at the right, to finish the missing 2 columns